StellarApp/Protocol/PlanetsMoved.swift

```diff
@@ -0,0 +1,190 @@
+                orbitNode.addChildNode(planetNode)
+                orbitNode.runAction(SCNAction.repeatForever(SCNAction.rotateBy(x: 0, y: 0.9, z: 0, duration: 1)))
+                planetNodes.append(orbitNode)
+                
+            case .saturn:
+                let orbitNode = SCNNode.getRings(radius: 9.42, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNodes(name: "saturn", image: UIImage(named: "art.scnassets/saturn.jpg")!, radius: 0.86, vector: (x: 9.42, y: 0, z: 0), moveSpeed: 0.7, rotation: 0.2)
+                let saturnOrbitNode = SCNNode.getSaturnRings(radius: 0.9, vector: (x: 0, y: 0, z: 0), texture: UIImage(named: "art.scnassets/saturnRing.png")!)
+                orbitNode.addChildNode(planetNode)
+                planetNode.addChildNode(saturnOrbitNode)
+                orbitNode.runAction(SCNAction.repeatForever(SCNAction.rotateBy(x: 0, y: 0.7, z: 0, duration: 1)))
+                planetNodes.append(orbitNode)
+                
+            case .uranus:
+                let orbitNode = SCNNode.getRings(radius: 12.58, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNodes(name: "uranus", image: UIImage(named: "art.scnassets/uranus.jpg")!, radius: 0.65, vector: (x: 12.58, y: 0, z: 0), moveSpeed: 0.5, rotation: 0.4)
+                orbitNode.addChildNode(planetNode)
+                orbitNode.runAction(SCNAction.repeatForever(SCNAction.rotateBy(x: 0, y: 0.5, z: 0, duration: 1)))
+                planetNodes.append(orbitNode)
+                
+            case .neptune:
+                let orbitNode = SCNNode.getRings(radius: 15.98, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNodes(name: "neptune", image: UIImage(named: "art.scnassets/neptune.jpg")!, radius: 0.5, vector: (x: 15.98, y: 0, z: 0), moveSpeed: 0.4, rotation: 0.8)
+                orbitNode.addChildNode(planetNode)
+                orbitNode.runAction(SCNAction.repeatForever(SCNAction.rotateBy(x: 0, y: 0.4, z: 0, duration: 1)))
+                planetNodes.append(orbitNode)
+                
+            case .pluto:
+                let orbitNode = SCNNode.getRings(radius: 18.65, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNodes(name: "pluto", image: UIImage(named: "art.scnassets/pluto.jpg")!, radius: 0.19, vector: (x: 18.65, y: 0, z: 0), moveSpeed: 0.4, rotation: 0.8)
+                orbitNode.addChildNode(planetNode)
+                orbitNode.runAction(SCNAction.repeatForever(SCNAction.rotateBy(x: 0, y: 0.4, z: 0, duration: 1)))
+                planetNodes.append(orbitNode)
+            }
+        }
+        return planetNodes
+    }
+}
+
+extension SCNNode {
+    static func deepScaleNodes(node: SCNNode, scaleAction: SCNAction) {
+        node.runAction(scaleAction)
```

Comment:
why is this extension in two files. Its the same function. Extensions should be in a folder called `Extensions` and the File name should be class/struct/obj + extension purpose, ex: `SCNNode+NodeBuilding.swift
`